rts/Sim/Misc/GlobalConstants.h

```diff
@@ -70,13 +70,34 @@ static constexpr int MAX_TEAMS = 255;
  */
 static constexpr int MAX_PLAYERS = 251;
 
+/**
+ * ID used to send chat messages to allies.
+ */
+static constexpr int TO_ALLIES = 252;
+
+/**
+ * ID used to send chat messages to spectators.
+ */
+static constexpr int TO_SPECTATORS = 253;
+
+/**
+ * ID send chat messages to everyone.
+ */
+static constexpr int TO_EVERYONE = 254;
+
+/**
+ * "player" number for GameServer-generated messages.
+ */
+static constexpr int SERVER_PLAYER = 255;
+
```

Comment:
the names should be self-explanatory enough not to split them up with "this is a bridge" style docs
```suggestion
static constexpr int MAX_PLAYERS   = 251;
static constexpr int TO_ALLIES     = 252;
static constexpr int TO_SPECTATORS = 253;
static constexpr int TO_EVERYONE   = 254;
static constexpr int SERVER_PLAYER = 255;
```